package_bootstrap/core/bootstrapper.py

```diff
@@ -0,0 +1,232 @@
+import sublime
+
+import os
+import re
+import codecs
+import textwrap
+from zipfile import ZipFile
+
+from threading import Thread
+
+from os.path import join, dirname, normpath, relpath
+from importlib import __import__ as do_import
+
+
+### ---------------------------------------------------------------------------
+
+
+# These name the bootstrap package that we use to give Sublime access to our
+# resources and commands, and the base name of the file within that package
+# that is responsible for loading the commands into Sublime when the package
+# is loaded.
+bootstrap_pkg = "1_my_package"
```

Comment:
probably the readme should mention more details about renaming this, and why it starts with a number :)

Reply:
The new commit below clarifies this in a new section in the top level README, explaining why it's named that way, how to change it, and that it's in your best interests to use something unique. There are also a bevy of other minor tweaks to word wrapping and extraneous spacing.  Along with the typos/changes you mentioned above, I appear to have pathologically spelled `bootstrapper` wrong about 5 times.